### PR TITLE
Show All Asset Types

### PR DESCRIPTION
**Pull request by LBHCallumM**
## Summary of Changes

- Remove AssetType restriction on AssetView page and EditAsset page
- AssetView component now conditionally displays cautionaryAlerts, TenureInformation and processes based on asset type
- Refactor AssetLayout component
- Refactor cautionary alerts implementation

### CODE REVIEW

File: src/views/asset-view/asset-view.tsx [hunk 1]
```diff
@@ -32,16 +35,17 @@ export const AssetView = (): JSX.Element => {
     );
   }
 
+  const showTenureInformation = isDwellingOrLettableNonDwelling(asset);
```
Comment (ange47rm) [inline src/views/asset-view/asset-view.tsx:38]: smart

Comment (LBHCallumM) [reply src/views/asset-view/asset-view.tsx:38]: Actually, I should refactor this. I repeat the line too many times

**Comment by ange47rm** [comment]
Good job refactoring a lot of that, looks much neater now.

I see yarn.lock has a few more entries. I was expecting it to only include a newer version of common (different commit hash), and no other changes. Do you know what the other yarn.lock changes are about?

Otherwise everything else looks good to me

File: src/components/asset-sidebar/index.test.tsx [hunk 2]
```diff
@@ -0,0 +1,157 @@
...
+      showTenureInformation
+    />,
+    {
+      url: `/property/${assetData.id}`,
+      path: "/property/:assetId",
+    },
+  );
+
+  // Assert
+  const tenureInformationHeading = screen.getByText("Tenure");
```
Comment (Asura5) [inline src/components/asset-sidebar/index.test.tsx:129]: Wouldn't the word Tenure also be visible on a component that said "New Tenure" ?

File: src/components/asset-sidebar/index.tsx [hunk 3]
```diff
@@ -45,16 +53,19 @@ export const AssetSideBar = ({ assetDetails, alerts, ...properties }: Props) =>
               {assetAddress.uprn ? "Edit address details" : "Cannot edit: UPRN missing"}
             </Button>
           )}
-          <CautionaryAlertsDetails alerts={alerts} />
-          <TenureDetails tenure={tenure} />
+          {showCautionaryAlerts && <CautionaryAlertsDetails alerts={alerts} />}
```
Comment (Asura5) [inline src/components/asset-sidebar/index.tsx:56]: Here you have a refactor which isn't necessarily part of the description - you're not showing the CC header if there are no Cautionary Contacts. is that correct?

**Review by Asura5 — APPROVED** [review]

**Pull request merged**